tools/docker/images/builder/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+
+ENV https_proxy=http://proxy-iil.intel.com:911
+ENV http_proxy=http://proxy-iil.intel.com:911
+ENV no_proxy=localhost,127.0.0.0/8,192.168.0.0/16,.intel.com,.lantiq.com
+ENV ftp_proxy=http://proxy-iil.intel.com:911
```

Comment:
This bit is definitely not something that should go to github :-)

From the docker-build man page:
> `--build-arg=variable`
>  name and value of a buildarg.
> 
>  For example, if you want to pass a value for http_proxy, use
>  `--build-arg=http_proxy="http://some.proxy.url"`
> 
> Users pass these values at build-time. Docker uses the buildargs as the
> environment context for command(s) run via the Dockerfile's RUN instruction
> or for variable expansion in other Dockerfile instructions. This is not meant
> for passing secret values. Read more about the buildargs instruction
> ⟨https://docs.docker.com/engine/reference/builder/#arg⟩